src/gui/screen_menu_steel_sheets.cpp

```diff
@@ -0,0 +1,177 @@
+#include "screen_menus.hpp"
+#include "gui.hpp"
+#include "app.h"
+#include "screen_menu.hpp"
+#include "i18n.h"
+#include "ScreenHandler.hpp"
+#include <type_traits>
+#include "Marlin/src/core/serial.h"
+#include "eeprom.h"
+#include "screen_sheet_rename.hpp"
+#include "wizard/screen_wizard.hpp"
+class ScreenMenuSteelSheets;
+
+enum class profile_action : uint32_t {
+    Select = 1,
+    Calibrate = 2,
+    Reset = 3,
+    Rename = 4
+};
+
+using sheet_index_0 = std::integral_constant<std::uint32_t, 0>;
+using sheet_index_1 = std::integral_constant<std::uint32_t, 1>;
+using sheet_index_2 = std::integral_constant<std::uint32_t, 2>;
+using sheet_index_3 = std::integral_constant<std::uint32_t, 3>;
+using sheet_index_4 = std::integral_constant<std::uint32_t, 4>;
+using sheet_index_5 = std::integral_constant<std::uint32_t, 5>;
+using sheet_index_6 = std::integral_constant<std::uint32_t, 6>;
+using sheet_index_7 = std::integral_constant<std::uint32_t, 7>;
+
+class MI_SHEET_SELECT : public WI_LABEL_t {
+    static constexpr const char *const label = N_("Select");
+
+public:
+    MI_SHEET_SELECT()
+        : WI_LABEL_t(label, 0, false, false) {};
+
+protected:
+    virtual void click(IWindowMenu &window_menu) override {
+        Screens::Access()->Get()->WindowEvent(nullptr, GUI_event_t::CHILD_CLICK, (void *)profile_action::Select);
+    }
+};
+
+class MI_SHEET_CALIBRATE : public WI_LABEL_t {
+    static constexpr const char *const label = N_("First layer cal.");
```

Comment:
may be we can skip the abbreviation, menu items should be able to roll text now
-> "First layer calibration" ... but please make sure it is not duplicated elsewhere